alerts/apps_alerts.libsonnet

```diff
@@ -60,6 +60,49 @@
             'for': '15m',
             alert: 'KubeDeploymentReplicasMismatch',
           },
+          {
+            alert: 'KubeDaemonSetRolloutStuck',
+            expr: |||
+              kube_daemonset_status_number_ready{%(kubeStateMetricsSelector)s}
+                /
+              kube_daemonset_status_desired_number_scheduled{%(kubeStateMetricsSelector)s} * 100 < 100
+            ||| % $._config,
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              message: 'Only {{$value}}% of desired pods scheduled and ready for daemon set {{$labels.namespace}}/{{$labels.daemonset}}',
+            },
+            'for': '15m',
+          },
+          {
+            alert: 'KubeDaemonSetNotScheduled',
+            expr: |||
+              kube_daemonset_status_desired_number_scheduled{%(kubeStateMetricsSelector)s}
+                -
+              kube_daemonset_status_current_number_scheduled{%(kubeStateMetricsSelector)s} > 0
+            ||| % $._config,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'A number of daemonsets are not scheduled.',
```

Comment:
Can we put `{{ labels.namespace }}/{{ labels.daemonset }}` in the message? With the current alert, the message will be repeated a bunch of times for each broken daemonset.